2022/22xxx/CVE-2022-23732.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
Expected location (2022/23xxx/CVE-2022-23732.json) of file does not match actual location (2022/22xxx/CVE-2022-23732.json)

Reply:
Updated to reflect correct directory ✅